# alpha_versioned_symbols.patch by Jordi Mallach <jordi@debian.org>
Index: alsa-lib-1.0.15/include/alsa-symbols.h
===================================================================
--- alsa-lib-1.0.15.orig/include/alsa-symbols.h	2007-10-20 21:47:11.000000000 +0200
+++ alsa-lib-1.0.15/include/alsa-symbols.h	2007-10-20 21:48:11.000000000 +0200
@@ -43,7 +43,7 @@
 	__asm__ (".symver " ASM_NAME(#real) "," ASM_NAME(#name) "@@" #version)
 #endif
 
-#ifdef USE_VERSIONED_SYMBOLS
+#if defined(USE_VERSIONED_SYMBOLS) || defined(__alpha__)
 #define use_symbol_version(real, name, version) \
 		symbol_version(real, name, version)
 #define use_default_symbol_version(real, name, version) \
